src/components/Settings/SettingDialog.tsx

```diff
@@ -123,3 +125,5 @@ export const SettingDialog: FC<Props> = ({ open, onClose }) => {
     </FloatingPortal>
   );
 };
+
+export default SettingDialog;
```

Comment:
Is it really needed change? Now it's harder to import settings dialog if I will need it somewhere else

Reply:
It is  not really needed. However, it is common practice:
 https://react.dev/learn/importing-and-exporting-components#default-vs-named-exports

Reverted back to the named export.